server/domain/feature.js

```diff
@@ -291,6 +316,11 @@ module.exports.updateFeatureToggle = function(applicationName, featureName, valu
     });
 };
 
+module.exports.updateFeature = function(applicationName, featureName, newFeatureDescription, req, cb){
+    addFeatureDescription(applicationName, featureName, newFeatureDescription);
+    cb();
```

Comment:
Still think this empty callback is a bit weird, the cb should be called once the description is changed.